client/state/posts/reducer.js

```diff
@@ -282,6 +281,23 @@ export function edits( state = {}, action ) {
 				[ action.siteId ]: omit( state[ action.siteId ], action.postId || '' )
 			} );
 
+		case POST_SAVE_SUCCESS:
+			if ( ! state.hasOwnProperty( action.siteId ) || ! action.savedPost ) {
```

Comment:
A little safety doesn't hurt, but is there ever a case where `action.savedPost` would be falsey?